test/unit/puppetlabs/services/jruby/jruby_puppet_testutils.clj

```diff
@@ -378,3 +379,30 @@ create-mock-pool-instance :- JRubyInstance
   (add-mock-jruby-pool-manager-service
    jruby-service-and-dependencies
    config))
+
+(defn copy-config-files
```

Comment:
I added pretty much this same code in a recent PR.  That's now [here|https://github.com/puppetlabs/puppetserver/blob/09c359c6343af06e5b7e9b667e94f61d54b82ce5/test/integration/puppetlabs/puppetserver/testutils.clj#L335-L368].  Would be good to rebase this PR and use the `with-config-dirs` instead of `with-puppet-config-files` macro.